scipy/signal/windows.py

```diff
@@ -1569,6 +1570,42 @@ def exponential(M, center=None, tau=1., sym=True):
     return w
 
 
+win_equiv_raw = {
```

Comment:
Would be good to prepend underscores to ``win_equiv_raw``, ``win_equiv``. ``needs_param``